website/content/api-docs/system/license.mdx

```diff
@@ -102,3 +116,113 @@ $ curl \
     --data @payload.json \
     http://127.0.0.1:8200/v1/sys/license
 ```
+
+## Delete License
+
+~> This API is deprecated and will be removed in a future version of Vault.
+Please use [license autoloading](/docs/enterprise/license/autoloading) instead.
+
+This endpoint is used to delete a stored license in Vault.  This will return
+an error unless license autoloading is in use.
+
+| Method | Path           |
+| :----- | :------------- |
+| `DELETE`  | `/sys/license` |
+
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request DELETE \
+    http://127.0.0.1:8200/v1/sys/license
+```
+
+## License Status
+
+This endpoint returns information about licensing.  It returns similar information
+to a GET of `sys/license`, but whereas that endpoint reports solely on the
+license Vault is currently actively using, this one reports on all licensing
+information the Vault cluster has.  See
+[license autoloading](/docs/enterprise/license/autoloading) for background.
+
+In the response:
+* `autoloading_used` will be true if an autoloaded license was provided to the
+  node, false otherwise
+* `autoloaded` is the autoloaded license if autoloading_used is true
+* `persisted_autoload` is the autoloaded license the active node last wrote to
+  storage; this is only used to detect inconsistently licensed nodes in the
+  cluster, it cannot be used as a license itself
+* `stored` is the stored license if any, i.e. one written using `PUT sys/license`
```

Comment:
I might end all of the above bullet points with a period.

Reply:
Done.